tests/keras_tests/feature_networks_tests/feature_networks/weights_mixed_precision_tests.py

```diff
@@ -189,10 +190,10 @@ def create_networks(self):
         return model
 
     def compare(self, quantized_model, float_model, input_x=None, quantization_info=None):
-        y = float_model.predict(input_x)
-        y_hat = quantized_model.predict(input_x)
-        cs = cosine_similarity(y, y_hat)
-        self.unit_test.assertTrue(np.isclose(cs, 1), msg=f'fail cosine similarity check:{cs}')
+        self.unit_test.assertTrue(isinstance(quantized_model.layers[2], DepthwiseConv2D))
+        self.unit_test.assertTrue(len(quantization_info.mixed_precision_cfg) == 1)
+        self.unit_test.assertTrue(quantization_info.mixed_precision_cfg[0] == 0) # Assert model is quantized using 16 bits as KPI is inf
```

Comment:
Is the max option here 16 bit? not 8?

Reply:
I think it's 16. We generate the TPC this way:
base_config = base_config.clone_and_edit(weights_n_bits=16,
                                                 activation_n_bits=16)
        mp_tp_model = generate_mixed_precision_test_tp_model(base_cfg=base_config,
                                                             mp_bitwidth_candidates_list=[(8, 16), (2, 16), (4, 16),
                                                                                          (16, 16)])